src/App.tsx

```diff
@@ -1,18 +1,12 @@
 import "./App.css";
```

Comment:
please, not include that file into the PR

Reply:
remove lessoner_project_frontend